core/shared/src/main/scala-2.12/plotly/MutableSequenceConversions.scala

```diff
@@ -0,0 +1,39 @@
+package plotly
+
+import plotly.Sequence.{DateTimes, Doubles, NestedDoubles, NestedInts, Strings}
+import plotly.element.LocalDateTime
+
+import scala.collection.immutable
+
+/**
+  * Provides implicit conversions from the possibly-mutable `scala.collection.Seq` to `plotly.Sequence`.
+  *
+  * This implementation is written for Scala 2.12 and does not leverage the efficient `.toSeq` conversion available
+  * since Scala 2.13. An equivalent implementation is provided for Scala 2.13 in the corresponding source root.
+  */
+trait MutableSequenceConversions {
+
+  private def makeImmutableSeq[A](mutableSeq: scala.collection.Seq[A]): immutable.Seq[A] =
+    mutableSeq match {
+      case asImmutableSeq: immutable.Seq[A] => asImmutableSeq
+      case _ => immutable.Seq[A](mutableSeq: _*)
+    }
```

Comment:
This is basically equivalent to the `.toSeq` or `scala.immutable.Seq.from` methods added in Scala 2.13 and used in the 2.13 implementation of this trait.